src/scss/helpers/variables.scss

```diff
@@ -0,0 +1,11 @@
+$primary-color: #3b0e8a;
+$primary-color-light: #c698f3;
+$primary-color-light-1: #7c7b7e;
+$primary-color-light-2: #e6e0f2;
+
+$primary-color-dark: #190934cc;
+
+$secondary-color: #f16600;
+
+$background-color-1: #fbf9fd;
+$text-color: #7f75a5;
```

Comment:
add variables in their own file( not need to create 2 variables files for scss... with the time it will be confusing where and in which file you will store variables)